core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2729,7 +2920,381 @@ components:
             data_struct:
               $ref: '#/components/schemas/DataStruct'
 
-
+#########################################
+# REQUEST: /lts/transaction/construction #
+#########################################
+    LtsTransactionConstructionRequest:
+      type: object
+      required:
+        - network
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+    LtsTransactionConstructionResponse:
+      type: object
+      required:
+        - epoch
+      properties:
+        epoch:
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 10000000000
+          description: An integer between `0` and `10^10`, marking the current epoch
+###################################
+# REQUEST: /lts/transaction/status #
+###################################
+    LtsTransactionStatusRequest:
+      type: object
+      required:
+        - network
+        - intent_hash
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        intent_hash:
+          $ref: "#/components/schemas/IntentHash"
+    LtsTransactionStatusResponse:
+      type: object
+      required:
+        - intent_status
+        - status_description
+        - known_payloads
+      properties:
+        intent_status:
+          $ref: "#/components/schemas/TransactionIntentStatus"
```

Comment:
It might be worth copying these down for `TransactionIntentStatus`/`TransactionPayloadStatus`, so that we allow ourselves to change these in the main transaction status endpoint in future if we want